SPECS/rpm/rpm.spec

```diff
@@ -285,6 +281,9 @@ popd
 %{python3_sitelib}/*
 
 %changelog
+* Thu Jun 15 2023 Andrew Phelps <anphel@microsoft.com> - 4.18.0-3
```

Comment:
Can't comment on the relevant line, but can we add a version constraint `>= 2.0-22` on the `mariner-rpm-macros` requirement in the `rpm-build` subpackage?

Reply:
sure, will add this